tests/console/yast2_ftp.pm

```diff
@@ -167,12 +168,13 @@ sub run {
     assert_screen 'yast2_ftp_create_upload_dir_confirm';    # confirm to create upload directory
     wait_screen_change { send_key 'alt-y' };
     assert_screen 'yast2_ftp_expert_settings';              # check passive mode value and enable SSL
-    wait_screen_change { send_key 'alt-m' };
+    wait_still_screen;                                      # wait until yast loads expert settings data
+    send_key 'alt-m';
     type_string_slow($vsftpd_directives->{pasv_min_port} . "\n");
-    wait_screen_change { send_key('alt-a', 1) };
+    send_key 'alt-a';
     type_string_slow($vsftpd_directives->{pasv_max_port} . "\n");
-    wait_screen_change { send_key('alt-l', 3) };            # enable SSL, and wait with next step
-    wait_screen_change { send_key 'alt-s' };                # give path for DSA certificate
+    wait_screen_change(sub { send_key 'alt-l'; }, 15);   # enable SSL, and wait with next step
```

Comment:
why specific timeout `15`?

Reply:
by default it was 10, enable ssl option has been skipped, therefore I have increased this value